xml_inscriptions/transcribed/UT.Provo.BYU.HBLL.G.2011.07.01.xml

```diff
@@ -0,0 +1,171 @@
+                            <rs/>  <!-- content can be "fake" or blank -->
+                        </summary>
+
+                        <origin><date notBefore="-0001" notAfter="0300">first - third centuries
+                                C.E.(?)</date>
+                            <placeName ref="asia.near_east">Palestine(?)</placeName>
+                        </origin>
+                        <provenance type="transferred" subtype="acquired" when-custom="1983-1986"> <p>From the <orgName>Royal Athena Gallery in New York</orgName> by a <persName role="acquier">Mr.
+                                    David H. Swingler</persName> of <orgName>Oxnard,
+                                    California</orgName> between <date notBefore="1982"
+                                    notAfter="1986">1983-1985</date></p></provenance>
+                        <provenance type="transferred" subtype="donated" when="2006"><p>y <persName role="donor">Mr. Swingler</persName> to the <orgName>Harold B. Lee Library at Brigham
+                                    Young University, Utah</orgName> in <date notBefore="2005"
+                                    notAfter="2007">2006</date></p></provenance>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#"/>
+                        <biblScope unit="page">ZPE_177_Blumell</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2020-09-16" who="Itamar Levin">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url=""/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                
+                <lb n="1"/>θάρσι <persName key="Heraklinos">Ἡρα
```

Comment:
`<persName><name key="Ἡρακλιανός">`  While we used to index using Romanized transcription, we now primarily use the Greek original.